webapp/src/components/connected_profiles.tsx

```diff
@@ -17,11 +17,11 @@ interface Props {
 const ConnectedProfiles = ({pictures, profiles, maxShowedProfiles, size, fontSize, border}: Props) => {
     maxShowedProfiles = maxShowedProfiles || 2;
     const diff = profiles.length - maxShowedProfiles;
+    const maxRightShift = 8 * maxShowedProfiles;
     profiles = diff > 0 ? profiles.slice(0, maxShowedProfiles) : profiles;
 
-    const margin = -8;
     const els = profiles.map((profile, idx) => {
-        const off = diff > 0 ? (profiles.length - idx) * 8 : (profiles.length - 1 - idx) * 8;
+        const rightShift = 8 * idx;
```

Comment:
I am thinking this is going to regress the spaning issue we've seen in https://github.com/mattermost/mattermost-plugin-calls/pull/11 :thinking:

Reply:
Ah, dammit. Well, we have to pick what we want then -- either:
1. The avatars shift slightly to the left horizontally in the single line post:
<img width="613" alt="image" src="https://user-images.githubusercontent.com/1490756/158677676-f54284af-8f2c-4998-900b-02fd3e2e724c.png">
vs
<img width="610" alt="image" src="https://user-images.githubusercontent.com/1490756/158677720-1c07b054-576f-490d-82eb-be4fb1cf548b.png">
vs
<img width="609" alt="image" src="https://user-images.githubusercontent.com/1490756/158677766-fd7265d6-9fa0-4cdd-98cc-11f2930e5fd2.png">

or:
2. The avatars shift slightly to the right in the double line post (they line up:
<img width="314" alt="image" src="https://user-images.githubusercontent.com/1490756/158677974-7c4d683b-461a-4dcf-b2bf-80d115df9799.png">
<img width="311" alt="image" src="https://user-images.githubusercontent.com/1490756/158678084-92b50fa7-675d-46b6-bd53-688fc40a05ba.png">
<img width="310" alt="image" src="https://user-images.githubusercontent.com/1490756/158678150-08082e9c-abb5-4cb8-9efe-65fdbfc2791f.png">

@abhijit-singh your choice!

Reply:
I think I prefer the horizontal shift (1), since it's easier to see the misalignment in (2).

Reply:
Yeah, with the current logic I don't think we can easily cover both cases. I agree that the vertical misalignment is far more noticeable than an extra margin.

Of course I am still hoping that there's an easier (and proper) way to do this kind of positioning and I just don't know how :p

Reply:
Brilliant, thanks @cpoile :tada: